static/app/views/settings/organizationAuditLog/index.tsx

```diff
@@ -85,6 +29,7 @@ class OrganizationAuditLog extends AsyncView<Props, State> {
           query: this.props.location.query,
         },
       ],
+      ['eventTypes', `/audit-log-api-names/`],
```

Comment:
Could we do this as an attribute on the auditlog entry API? Having a separate endpoint to fetch an effectively static list doesn't feel great, as it adds network overhead and latency.

If we can't modify the auditlog endpoint, could we use a build time operation to generate a JSON file that the frontend consumes? We do something similar for onboarding docs and translation files.